lib/project_types/script/messages/messages.rb

```diff
@@ -5,88 +5,87 @@ module Messages
     MESSAGES = {
       script: {
         help: <<~HELP,
-          Suite of commands for developing script applications. See {{command:%1$s script <command> --help}} for usage of each command.
+          Suite of commands for developing script applications. Run {{command:%1$s script <command> --help}} for usage of each command.
             Usage: {{command:%1$s script [ %2$s ]}}
         HELP
 
         error: {
-          deprecated_ep: "This project uses a Script API (%s) that has been deprecated. "\
-                         "This Script won't work in production.",
-          deprecated_ep_cause: "Try using a different Script API.",
+          deprecated_ep: "This script won't run in a store because "\
+                         "it uses a deprecated Script API (%s).",
+          deprecated_ep_cause: "Recreate this script using a supported Script API.",
           generic: "{{red:{{x}} Error}}",
           eacces_cause: "You don't have permission to write to this directory.",
-          eacces_help: "Try again and choose a different directory.",
+          eacces_help: "Get permission for this directory or choose a different one.",
 
           enospc_cause: "You don't have enough disk space to do this action.",
-          enospc_help: "Free up some space and try again.",
+          enospc_help: "Get more space.",
```

Comment:
This suggestion feels a little awkward to me because typically you free up space rather than getting more.

Reply:
Good call.